src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java

```diff
@@ -112,4 +122,19 @@ public AddClubPopUpStepOne clickCenterDropdown(){
         return this;
     }
 
+    @Step("Click on the dropdown with a list of centers")
+    public AddClubPopUpStepOne clickEditCenterDropdown(){
+        centerSelectedInEditClub.click();
+        return this;
+    }
```

Comment:
duplicate method, we have clickCenterDropdown for click

Reply:
Removed the duplicate, updated the clickCenterDropdown method